### PR TITLE
Our Team Page created and also fixed the footer issue of not sticking at bottom  

### PR DESCRIPTION
**Pull request by Vijay-Gunwant**
Fixes issue #42

### CODE REVIEW

**Comment by AdarshRawat1** [comment]
@Vijay-Gunwant Kindly add Screenshots or Screen recordings of the changes you have made.

File: css/style_footer.css
Comment (AdarshRawat1) [inline css/style_footer.css:1]: Avoid unnecessary changes to code base

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -2,9 +2,8 @@
 name: Feature request
 about: Suggest an idea for this project
 title: "[Feature Request]"
-labels: ''
-assignees: ''
-
+labels: ""
```
Comment (AdarshRawat1) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: Avoid making unnecessary changes.

File: css/team_list_style.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+#profile_img{
+    width: 8vw;
+    border-radius: 50%;
+}
+#social img{
+    width: 3vw;
+    margin: 2px;
+    border-radius: 50%;
+}
```
Comment (AdarshRawat1) [inline css/team_list_style.css:52]: Add a blank line at end of file.

File: data/Maintainers.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[
+    {
+    "name":"Adarsh Rawat",
+    "github":"https://github.com/AdarshRawat1",
+    "linkedin":"https://www.linkedin.com/in/adarsh-rawat/"
+    }
+]
```
Comment (AdarshRawat1) [inline data/Maintainers.json:7]: Add a new line here.

File: javascript/team_list.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      });
+  
+      // Set the HTML content of the 'card' element
+      const card = document.getElementById("cards_contributor");
+      card.innerHTML = str2;
+    })
+    .catch(error => {
+      console.error('Error fetching JSON:', error);
+    });
+
```
Comment (AdarshRawat1) [inline javascript/team_list.js:69]: add a new line here

File: team_list.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+            </div>
+            <div class="footer-rights">
+                Copyright &#169 opencodeera.pages.dev | All Rights Reserved
+            </div>
+        </footer>
+    </header>
+</body>
+<script src="javascript/team_list.js"></script>
+
+</html>
```
Comment (AdarshRawat1) [inline team_list.html:117]: Add a new line here

**Review by AdarshRawat1 — APPROVED** [review]

**Comment by AdarshRawat1** [comment]
@Vijay-Gunwant make few minor changes, and then we can merge your PR.

**Comment by AdarshRawat1** [comment]
> @Vijay-Gunwant Kindly add Screenshots or Screen recordings of the changes you have made.

@Vijay-Gunwant

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (Vijay-Gunwant) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: this change is being automatically added by vs code i tried to resolve it but vs code on ssving make it double quotes

File: css/team_list_style.css (hunk 2 above)
Comment (Vijay-Gunwant) [reply css/team_list_style.css:52]: ohk

File: data/Maintainers.json (hunk 3 above)
Comment (Vijay-Gunwant) [reply data/Maintainers.json:7]: ohk

File: javascript/team_list.js (hunk 4 above)
Comment (Vijay-Gunwant) [reply javascript/team_list.js:69]: ohk

File: team_list.html (hunk 5 above)
Comment (Vijay-Gunwant) [reply team_list.html:117]: ok

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (Vijay-Gunwant) [reply .github/ISSUE_TEMPLATE/feature_request.md]: i have resolved it

**Comment by Vijay-Gunwant** [comment]
> @Vijay-Gunwant Kindly add Screenshots or Screen recordings of the changes you have made.

Screenshot (83)
this is how the our teams page is looking

**Comment by Vijay-Gunwant** [comment]
> > @Vijay-Gunwant Kindly add Screenshots or Screen recordings of the changes you have made.
> 
> @Vijay-Gunwant

@AdarshRawat1 i have made the changes and shared the screen shot

**Comment by Vijay-Gunwant** [comment]
> @Vijay-Gunwant make few minor changes, and then we can merge your PR.

@AdarshRawat1 done

**Comment by AdarshRawat1** [comment]
@Vijay-Gunwant Thanks for the PR 🚀

**Review by s-dabral — COMMENTED** [review]
The code is good but avoid using too much lines between different section. Use comments to distinguish between the different sections of the project.

**Comment by Vijay-Gunwant** [comment]
> The code is good but avoid using too much lines between different section. Use comments to distinguish between the different sections of the project.

ohk i will take care of that
